package.json

```diff
@@ -180,6 +180,7 @@
     "localforage": "^1.7.1",
     "lodash-es": "^4.17.14",
```

Comment:
- [ ] Dev enhancement:  Is it possible to name the css files like we do for js?  `4706.8a77fbdb58c780694c7c.css` isn't helpful.

For js, the `webpackchunkname` was the only way to give the chunks a name. Not sure about css.

Reply:
Done.